### DISCUSSION TITLE
Magma quick start errors encountered

### DISCUSSION THREAD

**Discussion by jni2000** [magma/magma#14589] [Q&A] [answered]
Hello everyone. I just joined in the community and have some questions with regard to Magma quick start. Hope this is the place I can get answers. I have followed https://magma.github.io/magma/docs/basics/prerequisites
and I now go through: https://magma.github.io/magma/docs/basics/quick_start_guide

While following the quick start guide with Magma, I encountered some errors at both "vagrant up magma" and "make run" after ssh into magma.

At "vagrant up magma" stage, I see the following errors. Magma vbox did start, though.

fatal: [magma]: FAILED! => {"changed": true, "cmd": "rmmod openvswitch", "delta": "0:00:00.003840", "end": "2022-11-30 17:28:21.097661", "msg": "non-zero return code", "rc": 1, "start": "2022-11-30 17:28:21.093821", "stderr": "rmmod: ERROR: Module openvswitch is in use by: vport_gtp", "stderr_lines": ["rmmod: ERROR: Module openvswitch is in use by: vport_gtp"], "stdout": "", "stdout_lines": []}

fatal: [magma]: FAILED! => {"changed": false, "msg": "Unable to restart service networking: Job for networking.service failed because the control process exited with error code.\nSee \"systemctl status networking.service\" and \"journalctl -xe\" for details.\n"}

fatal: [magma]: FAILED! => {"changed": false, "checksum": "97481465e750436811ef39f5ea68836246e3db5f", "msg": "Destination directory /home/vagrant/build/python/lib/python3.8/site-packages/ryu/ofproto does not exist"}
-- this error still occurring  **even after I did "pip install ryu"**

At the "make run " stage, I see the following **conflict** errors about protobuf versions

ERROR: grpcio-tools 1.51.1 has requirement protobuf<5.0dev,>=4.21.6, but you'll have protobuf 3.20.1 which is incompatible.

ERROR: orc8r 0.0.0 has requirement protobuf==3.20.1, but you'll have protobuf 4.21.9 which is incompatible.

my original version was 3.20.1 and I did upgrade to 4.21.9 when I saw error the first time.Can anyone help me to understand the nature of such problems ? Thanks a lot.

**Comment by jni2000** [comment, marked as answer]
Resolved the problem by modifying the package dependencies with the required versions. No sure if this is the right way but at least resolved the problems "dirty and quick".

**Reply by giscard85** [reply]
Hello,
I'm having the same issue. Can you share your solution please?.

**Comment by panyogesh** [comment]
Closing the issue.
